site/templates/application/footer/social-media-buttons.jet

```diff
@@ -1,25 +0,0 @@
-{{block footerSocialMediaButtons()}}
```

Comment:
You can't delete this file... we don't know how many client templates rely on its existence.  You could totally move the contents of `social-media-icons.jet` into this file.